autocrop/cli.py

```diff
@@ -110,16 +121,17 @@ def main(
             print("No face detected: {}".format(reject_filename))
             reject_count += 1
         else:
-            output(input_filename, output_filename, image)
+            output(input_filename, output_filename, image, basename_noext, extension)
             print("Face detected:    {}".format(output_filename))
             output_count += 1
 
     # Stop and print status
-    print(
-        "{} input files, {} faces cropped, {} rejected".format(
-            input_count, output_count, reject_count
-        )
-    )
+
+    if extension is not None:
```

Comment:
I don't think adding extension info to the printout is valuable. Could you remove it please?

Reply:
Sure dev....